workshop/Lab0/README.md

```diff
@@ -112,7 +112,7 @@ functionality later. All the configuration files we use are under the directory
 
 Repo `kube101` contains the step by step instructions to run the workshop.
 
-```console
+```shell
 $ git clone https://github.com/IBM/guestbook.git
 $ git clone https://github.com/IBM/kube101.git
 ```
```

Comment:
nominal markdown linter default syntax will not accept a command example (as noted with the opening `$`) and no example output. If no example output is being provided, drop the `$` from the start of the lines to make copying easier.